internal/cypress/grep/grep.go

```diff
@@ -0,0 +1,47 @@
+// Package grep implements functions to parse and filter spec files by cypress-grep expressions.
+//
+// See https://www.npmjs.com/package/@cypress/grep for details on the specific syntax
+// of cypress-grep expressions.
+package grep
+
+import (
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/saucelabs/saucectl/internal/cypress/code"
+)
+
+// Match finds the files whose contents match the grep expression in the title parameter
+// and the grep tag expression in the tag parameter.
+func Match(rootDir string, files []string, title string, tag string) (matched []string, unmatched []string) {
+	for _, f := range files {
+		b, err := ioutil.ReadFile(filepath.Join(rootDir, f))	
```

Comment:
`ioutil` has been deprecated. Better to use `os.ReadFile` https://pkg.go.dev/os#ReadFile

Reply:
Good call. Ended up going with `fs.ReadFile` to make the function more testable.